src/components/simulation/SimulationEventsPanel.svelte

```diff
@@ -78,6 +78,14 @@
       resizable: true,
       sortable: true,
     },
+    span: {
```

Comment:
This field needs to be defined in the `SimulationEventColumns` type above. TS isn't too good at complaining about this for some reason...

Reply:
Got it I defined the span field in the `SimulationEventColumns`